mobile/docs/root/intro/version_history.rst

```diff
@@ -20,6 +20,7 @@ Breaking changes:
 - api: move ``dnsPreresolveHostnames`` APIs from taking concatenated cluster YAML to taking a list of String hostnames (:issue: `#25297 <25297>`, :issue: `#25259 <25259>`, :issue: `#25457 <25457>`)
 - api: added ``setRuntimeGuard`` APIs for all languages (:issue: `#25434 <25434>`)
 - api: added ``setRtdsLayer``, ``addAggregatedDiscoveryService``, ``setNodeId``, ``setNodeLocality`` APIs for all languages
+- api: added ``setCdsLayer`` APIs for all languages
```

Comment:
I continue to think that a separate nested xDS configuration object should be used so that we can enforce the expected grouping of values in the type system to make it impossible to miss one field that was required in a configuration. But not a blocker.

Reply:
Nit: missing issue

Reply:
Resolving the nit, and the other point is a good idea but yeah maybe for another PR